hints_tests.py

```diff
@@ -25,6 +25,11 @@ def test_program(program_name: str):
     # test_program("ec_doble")
     test_program("memcpy")
     test_program("memset")
+    #test_program("uint256_add")
+    #test_program("uint256_sqrt")
+    #test_program("uint256_unsigned_div_rem")
+    #test_program("uint256_signed_nn")
+    test_program("bigint_to_uint256")
```

Comment:
This test throws the error `ValueError: Custom Hint Error: ValueError: Failed to get ids value`, is that expected?